hazelcast-member.rb

```diff
@@ -0,0 +1,46 @@
+class HazelcastMember < Formula
+  desc "Command-line tool for running Hazelcast member instance(s)"
+  homepage "https://github.com/hazelcast/hazelcast"
+  url "https://github.com/hazelcast/hazelcast-member-tool/raw/master/archive/hazelcast-member-3.10.6.3.tar.gz"
```

Comment:
That's unusual… why is it not an official Github release?

Reply:
Good question, I'll change the url to a Github release.

Reply:
I have created an official Github release for it